workflow/rules/bwa-mem.smk

```diff
@@ -0,0 +1,48 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule bwa_mem:
+    input:
+        fastq=["Pre-alignment/merge/{sample}_R1.fq.gz", "Pre-alignment/merge/{sample}_R2.fq.gz"],
```

Comment:
I would be set fwd and rev as different stanza to make it more robust,
these should also be put in two lines I think according to snakefmt? It makes reading your code much easier

Reply:
The wrapper needs it this way. Otherwise we will have to skip the wrapper.
https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/bwa/mem.html

Reply:
I will take a look at the wrapper but maybe that is just the way it should be then. What about the unit wildcard? Is it possible to use it with the wrapper? Btw, how should we go about creating own wrappers?
I guess `snakefmt` should still be applied, right?

Reply:
One  could make change suggestion to the snakemake-wrapper repo, if we want to use fwd and rev as input